pkg/downloader/manager_test.go

```diff
@@ -168,3 +172,60 @@ func TestGetRepoNames(t *testing.T) {
 		}
 	}
 }
+
+func TestDownloadAllSubChartsOnly(t *testing.T) {
+	b := bytes.NewBuffer(nil)
+
+	//create a chart dir
+	tmp, err := ioutil.TempDir("", "helm-chart")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmp)
+
+	m := &Manager{
+		Out:       b,
+		HelmHome:  helmpath.Home("testdata/helmhome"),
+		ChartPath: tmp,
+	}
+	//create the /charts subdir
+	chartsFolder := filepath.Join(tmp, "charts")
+	if err != nil {
```

Comment:
what is the intent of checking here? Is this supposed to be after the call to `os.Mkdir`?

Reply:
indeed